assets/mods/simplify/postload.js

```diff
@@ -130,78 +151,59 @@
 	
 			return result;
 		}
+
+		/**
+		 * Given an ig.root-prefixed string, applies asset overrides.
+		 *
+		 * @param {string} oldpath
+		 * @returns {string} newpath
+		 */
+		_applyAssetOverrides(path) {
+			if (!path.startsWith(ig.root))
+				return;
+			const fullreplace = this._getAllAssets(path.substr(ig.root.length));
+	
+			if(fullreplace && fullreplace.length > 0){
+				if(fullreplace.length > 1)
+					console.warn('Conflict between \'' + fullreplace.join('\', \'') + '\' found. Taking \'' + fullreplace[0] + '\'');
+	
+				//console.log("Replacing '" + settings.url + "' with '" + fullreplace[0]  + "'");
+	
+				if (fullreplace[0].indexOf('assets') === 0) {
+					return ig.root + fullreplace[0].substr(7);
+				} else {
+					return ig.root + fullreplace[0];
+				}
+			}
+			return path;
+		}
 	
 		/**
 		 * 
 		 * @param {string} path 
 		 * @returns {Promise<string>}
 		 */
 		loadFilePatched(path) {
-			return new Promise((resolve, reject) => {
-				path = this._stripAssets(path);
-				const req = new XMLHttpRequest();
-				req.open('GET', path, true);
-				req.onreadystatechange = function(){
-					if(req.readyState === 4 && req.status >= 200 && req.status < 300) {
-						resolve(req.responseText);
-					}
-				};
-				req.onerror = err => reject(err);
-				req.send();
-			});
+			return this.loadFile(this._applyAssetOverrides(path));
```

Comment:
If a JSON file would be loaded using this function this would return incorrect results. Users cannot be trusted to use `loadJSONPatched` over `loadFilePatched`